.changeset/silver-geckos-smoke.md

```diff
@@ -0,0 +1,6 @@
+---
+"@replayio/playwright": patch
+"replayio": patch
+---
+
+Stuff
```

Comment:
```suggestion
```

Reply:
Looks like you beat me to this one 😉

Reply:
I had to temporarily add it to use `/release-pr` since the set of packages it publishes is derived from existing changesets. I really need to add some `--include-changed` or `--all` to the relevant Changesets command :v